### DISCUSSION TITLE
2024上半年研究速递 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#469] [General]
# 2024上半年研究速递 - Miao Yu | 于淼

https://yufree.cn/cn/2024/07/01/2024-research-paper/

**Comment by cxy0714** [comment]
所以说这种案例还有很多吗？所以南极谋杀案里流行感冒的Y疫苗导致寿命显著缩短的研究十年后才发现这个想法也早有渊源了吗。奥不过确实也蛮合理，评价长期效果的研究的确容易滞后吧，除非给大家都装上生理监测设备，每天报告结果及时发现异常。。。

对了，网站可以加上搜索功能吗，搜索整个博客文库？话说三水大哥是怎么迅速找到自己之前写过的文章呢🤣前几天我想到了什么现象（现在忘了，），记得应该在三水大哥什么博客里提过，但是找了一通没找到，发现了没有搜索的痛。

这种案例：
>这研究估计后面会被反复拿来当案例，丹麦的一处社会住房因室内空气中多氯联苯 (PCB) 超标被要求拆掉，研究人员给工人佩戴了腕带，然后测了所有209种PCBs，测到29个色谱峰，含量最多的三种却不在典型的七种PCBs指标或 WHO 12 PCB 中，这就很尴尬了，人为设定的优先控制污染物究竟控制了个啥？会不会之前构建在监测PCB单体上的暴露研究全都跑偏了？
推荐人：于淼

>链接：https://pubs.acs.org/doi/10.1021/acs.est.3c10304

>题目：Quantifying 209 Polychlorinated Biphenyl Congeners in Silicone Wristbands to Evaluate Differences in Exposure among Demolition Workers
>

**Reply by yufree** [reply]
额，说来惭愧，我自己也经常找不到之前提过的话题。不过用搜索引擎搜索关键词加上site: yufree.cn 应该就可以，我就算给网站加上搜索，本质也就是一个自定义站点的搜索引擎，我这种静态网页站点靠搜索引擎就够了。

**Reply by cxy0714** [reply]
嗷嗷，get了！
